perception/multi_object_tracker/src/tracker/motion_model/motion_model_base.cpp

```diff
@@ -46,16 +46,28 @@ bool MotionModel::predictState(const rclcpp::Time & time)
   if (dt < 0.0) {
     return false;
   }
-  // if dt is too large, shorten dt and repeat prediction
-  const uint32_t repeat = std::ceil(dt / dt_max_);
-  const double dt_ = dt / repeat;
-  for (uint32_t i = 0; i < repeat; ++i) {
-    if (!predictStateStep(dt_, ekf_)) {
-      return false;
-    }
-    // add interval to last_update_time_
-    last_update_time_ += rclcpp::Duration::from_seconds(dt_);
+  if (dt < 1e-6 /*1usec*/) {
+    return true;
   }
+
+  // // if dt is too large, shorten dt and repeat prediction
+  // {
+  //   const uint32_t repeat = std::floor(dt / dt_max_) + 1;
+  //   const double dt_ = dt / repeat;
+  //   for (uint32_t i = 0; i < repeat; ++i) {
+  //     if (!predictStateStep(dt_, ekf_)) {
+  //       return false;
+  //     }
+  //     // add interval to last_update_time_
+  //     last_update_time_ += rclcpp::Duration::from_seconds(dt_);
+  //   }
+  //   std::cout << "MotionModel::predictState predict  dt: " << dt << ", dt_: " << dt_  << ",
+  //   repeat:
+  //   "<< repeat<< std::endl;
+  // }
```

Comment:
Would you like to leave this comment out?

Reply:
the commented-out code is not necessary for updating SW.
I removed it.